src/MeshlessFV/MeshlessFVTree.cpp

```diff
@@ -578,7 +578,7 @@ void MeshlessFVTree<ndim,ParticleType>::UpdateGodunovFluxes
             neibmanager.GetParticleNeib(activepart[j],hydromask,do_pair_once);
 
 #if defined(VERIFY_ALL)
-        neibmanager.VerifyNeighbourList(i, Nhydro, mfvdata, "all");
+        neibmanager.VerifyNeighbourList(i, mfv->Nhydro, mfvdata, "all");
```

Comment:
In which case I'm sure it's fine. I only mentioned it because you'd made similar changes elsewhere. Were they really needed?